### PR TITLE
Add additional food icons

### LINKED ISSUE #1396: Meal tray
## Icon Request

* Icon name: Meal tray / `meal-tray`
* Use case: To represent bento or any kind of meal
* _Screenshots_ of similar icons: 
image
image

### PR DESCRIPTION
**Pull request by danielbayley**
Possibly closes #1396 and #1323. Related to #1289 and #1306.

### CODE REVIEW

File: icons/bowl-chopsticks.svg
Comment (karsa-mistmere) [inline icons/bowl-chopsticks.svg:1]: Looks super cute, but doesn't very consistent with salad bowl or soup bowl.

File: icons/bowl-overflow.svg
Comment (karsa-mistmere) [inline icons/bowl-overflow.svg:1]: Not sure about use cases for this, any ideas?

File: icons/meal-box.svg
Comment (karsa-mistmere) [inline icons/meal-box.svg:1]: Upside down grumpy robot?

File: icons/mortar-pestle.svg
Comment (karsa-mistmere) [inline icons/mortar-pestle.svg:1]: I think for this one I'd prefer something less isometric, especially compared to `bowl-chopsticks`, this feels more like `bowl-paddle` than a mortar and pestle.
mortar

File: icons/sushi-2.svg
Comment (karsa-mistmere) [inline icons/sushi-2.svg:1]: The centre is a bit too dense.

File: icons/toast.svg
Comment (karsa-mistmere) [inline icons/toast.svg:1]: I think the shape could be a bit more pronounced, e.g.:
toast

**Comment by karsa-mistmere** [comment]
I think three sushi icons are probably two too many, nonetheless, here's an extra from my drafts:
sushi
image

File: icons/sushi.svg
Comment (JanTrichter) [inline icons/sushi.svg:1]: @danielbayley  The center of this is pretty dense. If you scale the outer and inner rectangle out by by 1px on each side it has more space between the lines and still complies with the guide. Like this:

image

File: icons/toast.svg
Comment (JanTrichter) [reply icons/toast.svg:1]: Your suggestion looks pretty dense IMO. I think the original is pretty good although I kinda agree with our suggestion.

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**